CHANGELOG.rst

```diff
@@ -2,6 +2,14 @@
 Changelog for package rviz_visual_tools
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+4.1.4 (2023-01-05)
+------------------
+* Add arbitrary color option to publishCuboid (`#227 <https://github.com/PickNikRobotics/rviz_visual_tools/issues/227>`_)
+* Merge pull request `#228 <https://github.com/PickNikRobotics/rviz_visual_tools/issues/228>`_ from PickNikRobotics/pr-update-clang-format
```

Comment:
I accidentally merged instead of squash committed here. Can we just make the release notes say "Bump clang-format version to 14" and link to #228?

Reply:
Also. Can we get #226 merged as well?

It's failing CI because of the clang-format version, so that was fixed by the above.